styles.css

```diff
@@ -0,0 +1,1234 @@
+}
+
+.past h2 {
+  font-size: 20px;
+  color: #000;
+  text-align: center;
+  top: 1rem;
+  width: 90%;
+  height: 5vh;
+  margin: 0 auto;
+  margin-top: 1.5rem;
+  padding-top: 2.5rem;
+}
+
+.past .underline {
+  width: 15%;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  top: 1.2rem;
+  margin: 0 auto;
+  margin-top: 0.7rem;
+  margin-bottom: 1.2rem;
+  height: 2px;
+  background-color: orangered;
+}
+
+.hero {
+  width: 80%;
+  position: relative;
+  margin: 0.3rem auto;
+  text-align: center;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  background-image: url('images/conf1.jpg');
+  background-color: rgb(255, 69, 0, 0.9);
+  background-blend-mode: overlay;
+  background-position: center;
+  background-size: cover;
+  height: 45%;
+  opacity: 0.5;
+}
+
+.conf {
+  background-image: url('images/conf2.jpg');
+}
+
+.hero h3 {
+  color: white;
+  font-size: 18px;
+  margin: 0;
+}
+
+.see-whole {
+  display: none;
+}
+
+.partners {
+  display: none;
+}
+
+.mobile-partners {
+  background: rgb(62, 62, 62);
+}
+
+.mobile-partners .underline {
+  width: 15%;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  top: 0.3rem;
+  margin: 0 auto;
+  margin-top: 0.5rem;
+  margin-bottom: 1.2rem;
+  height: 2px;
+  background-color: orangered;
+}
+
+.mobile-partners h2 {
+  font-size: 20px;
+  color: #fff;
+  text-align: center;
+  top: 1rem;
+  width: 90%;
+  height: 5vh;
+  margin: 0 auto;
+  margin-top: 1.5rem;
+  padding-top: 0.1rem;
+}
+
+.partners h2 {
+  font-size: 20px;
+  color: #000;
+  text-align: center;
+  top: 1rem;
+  width: 90%;
+  height: 5vh;
+  margin: 0 auto;
+  margin-top: 1.5rem;
+  padding-top: 0.1rem;
+}
+
+.partners .underline {
+  width: 15%;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  top: 0.3rem;
+  margin: 0 auto;
+  margin-top: 0.5rem;
+  margin-bottom: 1.2rem;
+  height: 2px;
+  background-color: orangered;
+}
+
+.mobile-partners .partners-wrap {
+  height: 30%;
+  width: 90%;
+  margin: 3px auto;
+  display: flex;
+  flex-direction: row;
+}
+
+.part-cont img {
+  height: 8vh;
+  width: 20vw;
+  object-fit: contain;
+}
+
+.mobile-partners .partners-wrap img {
+  height: 80%;
+  object-fit: contain;
+  width: 25%;
+  margin: 5% auto;
+}
+
+.mobile-partners .partners-sub {
+  height: 30%;
+  width: 60%;
+  display: flex;
+  flex-direction: row;
+  margin: 3px auto;
+}
+
+.mobile-partners .partners-sub img {
+  height: 80%;
+  object-fit: contain;
+  width: 45%;
+  margin: 5% auto;
+}
+
+.mobile-partners .row .img {
+  width: 100%;
+  height: 100%;
+}
+
+.footer {
+  width: 100%;
+  background: #fff;
+  display: flex;
+  flex-direction: row;
+  height: 18vh;
+  justify-content: center;
+}
+
+.footer .container {
+  width: 100%;
+  position: relative;
+  bottom: 0;
+  left: 0;
+  display: flex;
+  margin: 2px auto;
+  color: #000;
+  background: #fff;
+  transform: translateY(-10px);
+}
```

Comment:
Great design @marvin-nyalik 🤝

- [ ] I would suggest you remove the black background from your logo and make sure it is well-aligned so that your design can emulate the design given.
- Your design
![Your mobile about footer](https://github.com/marvin-nyalik/SoccerFestival/assets/63917725/9c025230-56fe-445d-a758-846a87f5cec8)
- Given design
![Given mobile about footer](https://github.com/marvin-nyalik/SoccerFestival/assets/63917725/f5482af1-03f9-412a-a19b-fc66ebe4031c)